File: practices/Fizzbuzz.py
#CW  Fizzbuzz


for i in range(1,51):
    if i % 3 == 0 and i % 5 == 0: 
        print("FizzBuzz")
    elif i % 3 == 0: 
        print("Fizz")
    elif i % 5 == 0: 
        print("Buzz")
    else: 
        print(i)
"""statement = ""
for i in range(1,51):
    if i%3 == 0 or i%5 == 0:
        if i%3 == 0:
            statement + "Fizz"
        if i%5 == 0:
            statement + "Buzz"
    else:
        statement = i
    print(statement)
    statement = ""
"""""

File: C/practices/fizzbuzz.c
//CW Fizzbuzz
#include <stdio.h>

int main(){
    for(int count = 0; count <= 50; count++){
        if(count % 15 == 0){
            printf("FizzBuzz\n");
        } else if(count % 5 == 0 ){
            printf("Buzz\n");
        } else if(count % 3 == 0){
            printf("Fizz\n");
        } else{
            printf("%d\n", count);
        }
    }
}